lib/fencing/st_client.c

```diff
@@ -610,13 +610,8 @@ stonith_api_list(stonith_t * stonith, int call_options, const char *id, char **l
     rc = stonith_api_call(stonith, call_options, id, "list", NULL, timeout, &output);
 
     if (output && list_info) {
-        const char *list_str;
-
-        list_str = crm_element_value(output, F_STONITH_OUTPUT);
-
-        if (list_str) {
-            *list_info = strdup(list_str);
-        }
+        pcmk__copy_or_null(list_info,
```

Comment:
where api-list is used in pacemaker-code the return-pointer is initialized.
but it is a minimal change in how the api behaves. do we have a general guideline written somewhere?
as list has an explicit return-value one could use the returned pointer just in case of positive return-value and thus omit initialization.